pep-0700.rst

```diff
@@ -0,0 +1,110 @@
+PEP: 700
+Title: Additional Fields for the Simple API for Package Indexes
```

Comment:
Maybe a good idea to make clear in the title that these fields are for the new JSON form of the simple API, e.g. prepending "JSON" or "JSON form" to "Simple API"? Reading the title, I mistakenly assumed it was about the _HTML_ simple API, backporting things from the _new JSON [simple] API_, rather than adding these additional fields (despite seeing the discussion about this some time ago). But up to you of course, as the PEP author :slightly_smiling_face:

Reply:
Honestly, I originally had that but I removed it because I wanted the title to be reasonably short. It's still a bit too long for my liking...

Reply:
Ah, I see—good point. It's also got a double "for", which isn't ideal either.

If you want to shorten it, do you like this better? Its a fair bit shorter than the current, adds "JSON" and avoids the double "for":

```suggestion
Title: New Fields for the JSON Simple Package Index API
```

Or what about this? Adding "based" to be clearer/more readible, but still shorter than the current version:


```suggestion
Title: New Fields for the JSON-Based Simple Package Index API
```

Reply:
Actually, it's a (slightly) open question whether the fields get added to the HTML API as well, so I'm going to say no, let's leave it as it is. There are way more significant questions with the PEP content anyway 😉

Reply:
Totally fair :slightly_smiling_face: If you do want to shorten here, here are a couple options for you to consider:

```suggestion
Title: Additional Fields for the Simple Package Index API
```

or

```suggestion
Title: New Fields for the Simple Package Index API
```

Anyway, I'll stop nagging you now, sorry!